src/components/BaseComponents/FormGroup/FormGroup.tsx

```diff
@@ -25,8 +27,17 @@ export default function FormGroup({
   testProps = {}
 }) {
 
+  const {errorMsgs} = useContext(ErrorMsgContext);
+  const [errMessage,setErrorMessage] = useState(errorText);
+  useEffect(()=>{
+    const found = checkErrorMsgs(errorMsgs,id,name)
+    if(!found){
```

Comment:
I think this is my lack of understanding of the code... But why do we set an error if no error is 'found'?

Reply:
Yeah thats just field level error message we are updating.

Reply:
So is it updating after the error has been fixed?